src/main/java/com/pokegoapi/auth/GoogleAuthCallback.java

```diff
@@ -0,0 +1,8 @@
+package com.pokegoapi.auth;
```

Comment:
Then maybe without ignoring all lines but the opening and closing comment? Otherwise 👍